src/libraries/System.Numerics.Tensors/README.md

```diff
@@ -1,2 +1,3 @@
 # System.Numerics.Tensors
-This library has not been shipped publicly and is not accepting contributions at this time.
\ No newline at end of file
+
+Provides APIs for performing primitive operations over tensors represented by spans of memory.
```

Comment:
Would be good to update this README to the format that other libraries use, i.e. https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Configuration.Abstractions/README.md. This can happen in a follow-up but please don't forget about it.

Also, as this package is now shipping officially, we should add a PACKAGE readme as well. I will add an entry in https://github.com/dotnet/runtime/pull/91210 to track that work.